### PR TITLE
Cache Clear page

### PR DESCRIPTION
**Pull request by nate-li**

### CODE REVIEW

File: tinker/__init__.py [hunk 1]
```diff
@@ -70,13 +71,12 @@ def before_request():
...
         app.logger.debug("failed to init")
 
-
-@app.route('/cache-clear/<path:img_path>')
-@app.route('/cache-clear')
-def cache_test(img_path=None):
-    if not img_path:
-        img_path = '/academics/faculty/images/lundberg-kelsey.jpg'
-    return tools.clear_image_cache(img_path)
+# @app.route('/cache-clear/<path:img_path>')
```
Comment (e-jameson) [inline tinker/__init__.py]: @nate-li go ahead and just delete any old code if its no longer needed

File: tinker/admin/cache/templates/cache-home.html [hunk 2]
```diff
@@ -1,9 +1,38 @@
...
+        function submitpath() {
+            var urlpath = $("#cpath").val();
+
+            console.log("hello");
+            var input_data = {'url': urlpath};
+
+            var cacheurl = "/admin/cache-clear/submit";
+
+            $.post(cacheurl, input_data, function (data) {
+                alert("testing 234");
```
Comment (e-jameson) [inline tinker/admin/cache/templates/cache-home.html]: @nate-li can you remove this alert?

File: tinker/admin/cache/views.py [hunk 3]
```diff
@@ -9,4 +10,14 @@ def before_request():
...
+    return render_template('cache-home.html', **locals())
+
+@cache_blueprint.route('/submit', methods=['post'])
+def submit():
+    path = request.form['url']
+    return cache_clear(path)
+
+def cache_clear(img_path=None):
+    if not img_path:
+        img_path = '/academics/faculty/images/lundberg-kelsey.jpg'
```
Comment (e-jameson) [inline tinker/admin/cache/views.py]: @nate-li can we have a better case here if they don't submit a path? Either a check in the JS or a more helpful error message returned from Python?

**Comment by e-jameson** [comment]
Other than those 3 things this looks great!

**Comment by nate-li** [comment]
Fixed. Let me know if there's anything else that needs to be changed!

**Comment by e-jameson** [comment]
Can you clean it up a bit, so its more obvious its an error? Something like this:
image

**Pull request merged**